pkg/appstate/ingress.go

```diff
@@ -165,9 +166,22 @@ func ingressGetStateFromBackend(clientset kubernetes.Interface, namespace string
 }
 
 func ingressGetStateFromExternalIP(ing *networkingv1.Ingress) types.State {
-	lbIps := loadBalancerStatusIPs(ing.Status.LoadBalancer)
+	lbIps := ingressLoadBalancerStatusIPs(ing.Status.LoadBalancer)
 	if len(lbIps) > 0 {
 		return types.StateReady
 	}
 	return types.StateUnavailable
 }
+
+func ingressLoadBalancerStatusIPs(s networkingv1.IngressLoadBalancerStatus) sets.String {
```

Comment:
this is a copy of `loadBalancerStatusIPs`, but it's because of the newly introduced `networkingv1.IngressLoadBalancerStatus` type.